charts/tekton-triggers/templates/interceptors.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-triggers-core-interceptors-certs
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/name: core-interceptors
+    app.kubernetes.io/component: interceptors
```

Comment:
please split this file to corresponding file names
e.g.: ConfigMap to configmap.yaml